.env.development

```
@@ -11,4 +11,4 @@ NEXT_PUBLIC_ADMIN_API_URL=https://api-dev.sopt.org/v2
 NEXT_PUBLIC_ADMIN_API_KEY=
 NEXT_PUBLIC_OPERATION_API_URL=https://operation.api.dev.sopt.org
 NEXT_PUBLIC_AUTH_API_URL=https://auth.api.dev.sopt.org
-NEXT_PUBLIC_CREW_API_URL=https://crew.api.dev.sopt.org
\ No newline at end of file
+NEXT_PUBLIC_CREW_API_URL=https://crew-api-dev.sopt.org/
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)